readthedocs/proxito/views/hosting.py

```diff
@@ -68,12 +77,28 @@ def get(self, request):
                     "enabled": True,
                     "query_selector": "[role=main]",
                     "versions": list(
-                        project.versions.filter(active=True).values_list(
-                            "slug", flat=True
-                        )
+                        project.versions.filter(active=True)
+                        .only("slug")
+                        .values_list("slug", flat=True)
                     ),
                 },
-                "doc_diff": True,
+                "doc_diff": {
+                    "enabled": True,
+                    # "http://test-builds-local.devthedocs.org/en/latest/index.html"
+                    "base_url": f"""{resolver.resolve(
+                        project=project,
+                        version_slug=project.get_default_version(),
+                        language=project.language,
+                        filename=unresolved_url.filename,
+                    )}""",
```

Comment:
he he, yeah. This is terrible code 😝 -- It was on a f-string because it used to have other replacement in the string itself, but now it's just one function. So, the f-string is not required.

I think I will create helper methods to calculate these stuffs so it's more clear where they come from and easy to read. There is lot to iterate over in this code 😅